Geometry/HGCalCommonData/data/hgcal/v9/hgcal.xml

```diff
@@ -42,32 +42,33 @@
   <Constant name="radMixL13"             value="902.0*mm"/>
   <Constant name="radMixL14"             value="902.0*mm"/>
   <Constant name="radMixL15"             value="902.0*mm"/>
+  <Constant name="slope1"                value="[etaMax:slope]"/>
```

Comment:
@bsunanda - there isn't much point in introducing an intermediate constant.

Reply:
We are planning alternative version of geometry. These changes are made in view of that. Also in future we would deviate from etaMax:slope